deploy/common/k8sbased.sh

```diff
@@ -631,6 +650,7 @@ function launch_sensor {
 
       if [[ "$SENSOR_SCANNER_SUPPORT" == "true" ]]; then
         helm_args+=(--set scanner.disable=false)
+        helm_args+=(--set scannerV4.disable=false)
```

Comment:
is it ok to sneak this new value in with this PR? (there is no other mention of Scanner V4)

Reply:
Right, sorry, about that. Will remove it.